schedule/virtualization/vmware_hyperv.yaml

```diff
@@ -9,3 +9,4 @@ schedule:
     - virtualization/xen/ssh_guests_init
     - virtualization/xen/upgrade_guests
     - virtualization/xen/patch_guests
+    - virtualization/xen/open_vm_tools
```

Comment:
Right now this file is used for both VMware but also Hyper-V. Do we wanna to run this also on Hyper-V?

Reply:
Isn't `open-vm-tools` only for vmware? In that case I guess we need a condition to run it only on vmware